module/connection/DatabaseDialog.kt

```diff
@@ -103,15 +103,28 @@ object DatabaseDialog {
     }
 
     @Composable
-    private fun DeletableDatabaseList(modifier: Modifier) = ActionableList.Layout(
+    private fun ManageDatabaseList(modifier: Modifier) = ActionableList.Layout(
         items = Service.client.databaseList,
         modifier = modifier.border(1.dp, Theme.studio.border),
         buttonSide = ActionableList.Side.RIGHT,
-        buttonFn = { databaseName ->
-            IconButtonArg(
-                icon = Icon.DELETE,
-                color = { Theme.studio.errorStroke },
-                onClick = {
+        buttonsFn = { databaseName ->
+            listOf(
+                IconButtonArg(
+                    icon = Icon.EXPORT,
+                    enabled = Service.project.current != null && !Service.schema.hasRunningCommand,
+                    tooltip = Tooltip.Arg(title = Label.EXPORT_SCHEMA)
+                ) {
+                    Service.client.fetchSchema(databaseName).let { schema ->
+                        Service.project.tryCreateUntitledFile()?.let { file ->
+                            file.content(schema ?: "")
```

Comment:
In the unlikely event that the database has been deleted between the list being displayed and the button being clicked, we'll display an empty file. Would it be preferable if the button did nothing in this case?

Reply:
To answer this, we'd have to ask what happens to the rest of studio interface areas when the database is deleted?

Reply:
If we have a session open with a database when it's deleted, the session ends and the user is warned that this has happened. 

However, we may not have a session open with any database when we see this list. In any case, it's a certainty that a scenario will always exist where a user clicks on this button to export the schema after the database is deleted. I think displaying a warning notification refreshing the database list in the event that the schema is null is probably safest.